vowpalwabbit/core/include/vw/core/confidence_sequence_robust.h

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#pragma once
+
+#if !defined(__APPLE__) && !defined(_WIN32)
+#  define __STDCPP_MATH_SPEC_FUNCS__ 201003L
+#  define __STDCPP_WANT_MATH_SPEC_FUNCS__ 1
+#endif
+
+#include "vw/core/io_buf.h"
+#include "vw/core/metric_sink.h"
+
+constexpr double CS_ROBUST_DEFAULT_ALPHA = 0.05f;
+
+namespace VW
+{
+
+// Equivalent to GTilde in confidence_sequence_robust.py
+class g_tilde
+{
+public:
+  g_tilde(double k);
+  double histo_variance(double lam_sqrt_tp1) const;
+  void histo_insert(double x);
+  void add_obs(double x);
+  double get_s() const;
+  double get_v(double lam_sqrt_tp1) const;
+  void reset_stats();
+
+  const double k;
+  const double log_k;
```

Comment:
Const means this class cannot be moved. Is that what is indended?

Reply:
I think the const needs to be removed